db/migrate/20230522194044_move_concurrent_requests_settings_to_vmware.rb

```diff
@@ -0,0 +1,40 @@
+class MoveConcurrentRequestsSettingsToVmware < ActiveRecord::Migration[6.1]
+  class SettingsChange < ActiveRecord::Base
+    serialize :value
+  end
+
+  def up
+    # capture_batch_size
+    #   introduced d1b21ac5444ab07c 10/2022
+    #   only currently populated for vmware
+    #
+    # merged into concurrent_requests
+    #
+    # rename "/ems/ems_*/capture_batch_size" =>
+    #        "/ems/ems_*/concurrent_requests/realtime"
+    # update_all("key = replace(key, 'capture_batch_size', 'concurrent_requests/realtime')")
+    say_with_time("Removing old capture_batch_size keys") do
+      SettingsChange.where("key LIKE ?", "/ems/ems_vmware/capture_batch_size")
+                    .destroy_all
+    end
```

Comment:
@kbrock at the point this schema change will run we won't have any settings keys like `/ems/ems_vmware/capture_batch_size` right?

I think we only need to rename `/performance/concurrent_requests/*` to `/ems/ems_vmware/concurrent_requests/*` because vmware was the only provider using this.

Reply:
ok, removed. thanks